sdks/java/maven-archetypes/starter/pom.xml

```diff
@@ -60,6 +60,12 @@
               <goals>
                 <goal>integration-test</goal>
               </goals>
+              <configuration>
+                <!--
+                On windows EOL will be \n\r but we expect \n in tests.
+                -->
+                <ignoreEOLStyle>true</ignoreEOLStyle>
```

Comment:
maven-archetype-plugin runs some tests comparing to src/test resources the generated files - see http://maven.apache.org/archetype/maven-archetype-plugin/integration-test-mojo.html for the details. It is src/test/resources/projects/basic/reference in beam case. These files was using unix style EOL for me so needed this option.


Side note I forgot to mention: beam doesn't enforce EOL in gitattributes, maybe it would be good to do so?

I usually use as default:

    autocrlf = input
    eol = lf


wdyt?